infocmdb/infocmdb.go

```diff
@@ -36,7 +38,26 @@ type Client struct {
 	v2 *v2.Cmdb
 }
 
+type logOutputSplitter struct{}
+
+func (splitter *logOutputSplitter) Write(msg []byte) (n int, err error) {
+	if bytes.HasPrefix(msg, []byte("[WARNING]")) ||
+		bytes.HasPrefix(msg, []byte("[ERROR]")) ||
+		bytes.HasPrefix(msg, []byte("[FATAL]")) ||
+		bytes.HasPrefix(msg, []byte("[PANIC]")) {
+		return os.Stderr.Write(msg)
+	}
+	return os.Stdout.Write(msg)
+}
```

Comment:
Please switch it to check if it is a regular Stdout output if not print all to Stderr.

Reply:
Done, please re-review the "configures log output" commit.